src/components/App/App.js

```diff
@@ -9,18 +9,47 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      current: 'login'
+      current: 'login',
+      areas: []
     }
   }
 
+  componentDidMount() {
+    fetch('https://vrad-api.herokuapp.com/api/v1/areas')
+      .then(response => response.json())
+      .then(areaData => {
+        const areaPromises = areaData.areas.map(area => {
+          return fetch(`https://vrad-api.herokuapp.com${area.details}`)
+            .then(response => response.json())
+            .then(info => {
+              return {
+                area: area.area,
+                details: area.details,
+                id: info.id,
+                name: info.name,
+                location: info.location,
+                about: info.about,
+                region_code: info.region_code,
+                quick_search: info.quick_search,
+                listings: info.listings
+              }
+            })
+        })
+        console.log(areaPromises);
+        Promise.all(areaPromises)
+          .then(completeAreaData => this.setState({areas: completeAreaData}))
+      })
+      .catch(err => console.error(err))
+  }
 
   render() {
     return (
       <div className="App">
-        <Login />
+        {/*<Login />*/}
```

Comment:
Commented this out just so I could focus on styling for the areasContainer - tomorrow we will figure out how exactly to render the right page when.

Reply:
I have the login functionality ready to go to display correct info on page change as well